package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-verisure",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Sure thing. I'm not sitting with the machine with git at the moment so feel free to do the changes 👍